### PR TITLE
Mount /etc/localtime on macOS with Orbstack

### PR DESCRIPTION
**Pull request by cHJlaXpoZXI**
# Description

This PR is to delete line that block Orbstack to mount /etc/localtime. Orbstack on macOS can mount this folder proprely now. I have tested several time (no option, VPN and Desktop) and no issue, the time is sync with the host !

# Related issues

No related issue

# Point of attention

Hope that my modification is correct :)

### CODE REVIEW

**Comment by ShutdownRepo** [comment]
@QU35T-code can you confirm?

**Comment by Dramelac** [comment]
Thank you for the PR. We need to do some testing before merging.

Regarding your code, my only concern is because of the docker-desktop patch: 
`path_match = path_match.replace("/etc/", "/private/etc/")`

Not sure if it's necessary when using orbstack or not

**Comment by cHJlaXpoZXI** [comment]
> Thank you for the PR. We need to do some testing before merging.
> 
> Regarding your code, my only concern is because of the docker-desktop patch: `path_match = path_match.replace("/etc/", "/private/etc/")`
> 
> Not sure if it's necessary when using orbstack or not

Orbstack mount the /etc/localtime by default. So I guess no. I haven't delete it because I don't know if this was used somewhere else or not.

If not, we can remove everything then.

And sure for testing no problem ;) if I can contribute with test, tell me ;)

File: exegol/model/ContainerConfig.py [hunk 1]
```diff
@@ -963,8 +963,6 @@ def addVolume(self,
                         logger.critical(msg)
                     raise CancelOperation(msg)
                 if path_match.startswith("/etc/"):
```
Comment (Dramelac) [inline exegol/model/ContainerConfig.py:965]: Change to `if path_match.startswith("/etc/") and EnvInfo.isDockerDesktop():`

Comment (cHJlaXpoZXI) [reply exegol/model/ContainerConfig.py]: I make the change :)

**Comment by Dramelac** [comment]
@ShutdownRepo or @QU35T-code Can you confirm Orbstack can bind volume from `/etc` ?
Can you also test with Docker Desktop if it works or not ?

**Review by Dramelac — APPROVED** [review]
Thank you for the testing and the PR !

**Pull request merged**